source/banner.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="banner">
+  <div class="container">
+    <h1 class="banner-logo">
```

Comment:
I don’t think this needs to be wrapped in an `h1`. Rather, each page’s unique title should be that page’s one `h1`. Think of the `h1` as the label for each page, and if every page was “Refills”, it would be rather repetitive. So in this case, what you have an as `h2` below, could possible become the `h1`.

I’ve been reading “[Inclusive Front-End Design Patterns](https://www.smashingmagazine.com/inclusive-design-patterns/)” and it breaks down how to structure headings, really nicely.